python/mxnet/test_utils.py

```diff
@@ -1203,10 +1201,10 @@ def check_speed(sym, location=None, ctx=None, N=20, grad_req=None, typ="whole",
     else:
         raise ValueError('typ can only be "whole" or "forward".')
 
-
 def check_consistency(sym, ctx_list, scale=1.0, grad_req='write',
                       arg_params=None, aux_params=None, tol=None,
-                      raise_on_err=True, ground_truth=None, equal_nan=False, use_uniform=False):
+                      raise_on_err=True, ground_truth=None, equal_nan=False,
+                      use_uniform=False, default_type=np.float64):
```

Comment:
Rename the argument 'default_type' to 'data_type'?

Reply:
or random_type?
since its associated with the data type for the randomly generated data? If the caller passes in arg_params then this argument is ignored